src/styles/nav.css

```diff
@@ -1,5 +1,10 @@
 .c-nav-link {
 	padding: 0 1rem;
+  pointer-events: none;
```

Comment:
Nit: A tab is used to indent just above, could you convert it to spaces ?

Reply:
See https://github.com/cozy/cozy-passwords/pull/1/commits/eb59c978a54d3e0aab6bca2d3b2943c7ec8c6a1c